resources/views/navbar/profile.blade.php

```diff
@@ -10,6 +10,7 @@
             @isset($identifier)
                 <x-ark-avatar
                     :identifier="$identifier"
+                    :identifier-with-letters="$identifierWithLetters ?? false"
```

Comment:
make it `use-identifier-letters` to indicate it's using something instead of making it look like an additional identifier option that takes letters

Reply:
or maybe `show-identifier-letters` would be even better 🤔

Reply:
this hasn't been done yet @Highjhacker